com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -721,14 +721,6 @@ private void Awake()
 
         private void OnEnable()
         {
-            if (Singleton != null && Singleton != this)
-            {
-                Destroy(gameObject);
```

Comment:
I'm not super familiar with what this code does, but when I read the PR title I anticipated to see "if multi-scene, don't destroy, else still do" but what I see here is we don't destroy the game object anymore

Reply:
Yeah, it makes no sense for us to destroy it. I can't remember why I did it that way.

Basically what this used to do was:

If there was a NetworkManager already, destroy myself.

Now it doesn't destroy it anymore. That's the only difference.